src/cmdlib.sh

```diff
@@ -384,6 +414,18 @@ EOF
     else
         rm -vf "${local_overrides_lockfile}"
     fi
+    if [ -f /srv/redhat-coreos/content_sets.yaml ]; then
+        create_content_manifest
+    fi
+    if [ -f "${workdir}/tmp/buildinfo/content_manifest.json" ]; then
+        mkdir -p "${tmp_overridesdir}"/contentsetrootfs/usr/share/buildinfo/
+        jq . "${workdir}/tmp/buildinfo/content_manifest.json" > "${tmp_overridesdir}/contentsetrootfs/usr/share/buildinfo/content_manifest.json"
+        echo -n "Committing ${tmp_overridesdir}/contentsetrootfs... "
+        ostree commit --repo="$tmprepo" --tree=dir="${tmp_overridesdir}"/contentsetrootfs -b contentset 
+        cat >> "${override_manifest}" << EOF
+  - contentset
```

Comment:
This needs to go under `ostree-layers`. So we should probably instead just add it to the `layers` variable and then move down the block that outputs `ostree-layers` to after this.